mainapp/management/commands/create_bot.py

```diff
@@ -0,0 +1,17 @@
+from model_bakery import baker
+from mainapp.models import Bot
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Create random bots'
+
+    def add_arguments(self, parser):
+        parser.add_argument('-t', '--total', type=int, default=1, help='Indicates the number of bots to be created')
+
+    def handle(self, *args, **options):
+        total = options['total']
+        for i in range(total):
```

Comment:
```suggestion
        for _ in range(total):
```
convention is to use _ for variables we are not going to use